util/topgen.py

```diff
@@ -957,6 +957,11 @@ def main():
         type=int,
         metavar='<seed>',
         help='Custom seed for RNG to compute netlist constants.')
+    # Miscellaneous: only return the list of blocks and exit.
+    parser.add_argument('--get_blocks',
```

Comment:
@eunchan @tjaychen @msfschaffner @rswarbrick 
PTAL at this commit. It will be nice to be able to query stuff from topgen like this, rather than use it to exclusively auto generate stuff. The result of the query can be used in other scripts (like the following commit).